src/yb/client/client-internal.cc

```diff
@@ -1273,10 +1273,13 @@ Status YBClient::Data::RemoveMasterAddress(const HostPort& addr) {
     if (it != master_server_addrs_.end()) {
       master_server_addrs_.erase(it);
     }
-  }
 
-  RETURN_NOT_OK(SetMasterAddresses(HostPort::ToCommaSeparatedString(new_list)));
+    for(const auto& a : master_server_addrs_) {
+        updated_addrs += a;
```

Comment:
Are we missing commas (",") here when concatenating. Previously, we were doing ToCommaSeparatedString?

Reply:
@kmuthukk ya I missed  ',' separator. Do we have a way to do unit testing so that i can avoid these trivial errors?

Reply:
@kmuthukk updated